apps/prairielearn/elements/pl-file-preview/pl-file-preview.css

```diff
@@ -3,7 +3,6 @@
 }
 
 [id^='file-preview'] .file-status-icon {
-  margin-right: 10px;
   transition: all 400ms;
 }
```

Comment:
It seems you removed this class, is it still needed?

Reply:
Good call, it is not! Removed in 56c7603.